docs/tableADCP_sub_group.adoc

```diff
@@ -0,0 +1,144 @@
+
+ |{var}float transmit_duration_nominal_sub_pulse(ping_time) |MA |Nominal duration of the transmit sub-pulse. This is in the case where the transmitted pulse consists of a series of sub-pulses. ADCP specific.
+ 3+|{attr}:units = "s" 
+ 3+|{attr}:long_name = "Nominal duration of the transmitted sub-pulse"
+ 3+|{attr}float :valid_min = 0.0 
+
+ |{var}float transmit_lag_interval_sub_pulse(ping_time) |MA |Lag interval between the transmitted sub-pulse.This is in the case where the transmitted pulse consists of a series of sub-pulses. ADCP specific.
+ 3+|{attr}:units = "s" 
+ 3+|{attr}:long_name = "Lag interval of the transmitted sub-pulse"
+ 3+|{attr}float :valid_min = 0.0 
+
+ |{var}int quality(ping_time) |MA |Quality percent for each depth cell.
+ 3+|{attr}:units = "%" 
+ 3+|{attr}:long_name = "Quality indicator for the water current velocity calculation" 
+ 3+|{attr}int :valid_range = 0, 100 
+
+ |{var}float scaling_factor |MA |Scaling factor from ADCP calibration used in velocity calculations.
+ 3+|{attr}:long_name = "Scaling factor for velocity calculations" 
+ 3+|{attr}float :valid_range = 0.0, 2.0 
+
+ |{var}float slant_range_to_bottom(ping_time, beam) |MA |Detected bottom in each beam.
+ 3+|{attr}:units = "m" 
+ 3+|{attr}:long_name = "Slant range to bottom for each beam" 
+
+ |{var}int sv_dbw_high_limit(ping_time) |MA |Filtering parameter. Element discarded if backscatter is above given limit.
+ 3+|{attr}:units = "dB" 
+ 3+|{attr}:long_name = "Sv maximum limit used for filtering current ping" 
+ 3+|{attr}int :valid_range = -235, 0 
+
+ |{var}int sv_dbw_low_limit(ping_time) |MA |Filtering parameter. Element discarded if backscatter is below given limit.
+ 3+|{attr}:units = "dB" 
+ 3+|{attr}:long_name = "Sv minimum limit used for filtering current ping" 
+ 3+|{attr}int :valid_range = -235, 0 
+
+ |{var}sample_v velocity(ping_time, beam) |MA |Calculated beam velocity. Each element in the 2D matrix is a variable length vector that contains the calculated velocity along that beam and ping time.
+ 3+|{attr}:units = "m/s" 
+ 3+|{attr}:long_name = "Calculated beam velocity"
+ 3+|{attr}float :valid_range = -50.0, 50.0 
+
+ |{var}beam_stabilisation_t velocity_depth_stabilisation(ping_time) |MA |Indicates whether samples used for velocity vector calculation have been compensated for platform motion.
+ 3+|{attr}:long_name = "Velocity depth stabilization applied (or not) "
+
+ |{var}beam_stabilisation_t velocity_motion_stabilisation(ping_time) |MA |Indicates whether beam velocities have been compensated for platform motion.
+ 3+|{attr}:long_name = "Velocity motion stabilization applied (or not) "
+
+ |{var}float vertical_sample_interval(ping_time) |M |Distance between samples.
+ 3+|{attr}:units = "s" 
+ 3+|{attr}:long_name = "Interval between recorded data samples" 
+ 3+|{attr}float :valid_min = 0.0 
+
+e|Subgroups | |
+ |{var}Mean_current |O |Calculated averaged current velocities.
+  3+|{attr}:long_name = "Sonar/Beam_group1/ADCP/Mean_current" 
```

Comment:
This should be subgroup description 

Something like this

|Subgroups | |
 |{var}Mean_current |O | Subgroup containing calculated averaged current velocities data.